app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :expenditures
+
+  has_many :groups
+end
```

Comment:
- [ ]  Please add validation in your model
  
  `` validates :email, uniqueness: true, length: { in: 5..30 }``
  `` validates :username, uniqueness: true, length: { in: 2..30 }``
  
  For more information, read more about the association in rails [link1](https://guides.rubyonrails.org/association_basics.html) and [link2](https://medium.com/swlh/the-simplest-way-to-understand-associations-in-rails-dc03cfe067f2). Your models need other associations